tesseract_core/sdk/tesseract.py

```diff
@@ -23,20 +22,6 @@
 PathLike = str | Path
 
 
-@dataclass
```

Comment:
Out of curiosity, why should we remove this, instead of `**SpawnConfig.asdict()` in the `serve` call?

(not important, imho dict is fine as well)

Reply:
Just one more thing to keep in sync when modifying signatures that doesn't really add anything (since we're just converting it to dict right away vs. relying on types of fields etc.)